docs/api/topics/feed.rst

```diff
@@ -891,9 +904,13 @@ Create
         :ref:`app list <feed-collections-grouped>`.
     :param background_image_upload_url: a URL pointing to an image
     :type background_image_upload_url: string
-    :param color: a hex color used in display of the collection. Currently must
-        be one of ``#B90000``, ``#FF4E00``, ``#CD6723``, ``#00AACC``,
-        ``#5F9B0A``, or ``#2C393B``.
+    :param background_color: [DEPRECATED] a hex color used in display of the
```

Comment:
Is there a way to use the [`:deprecated:`](http://pythonhosted.org/sphinxcontrib-httpdomain/#options) marker here?